bot.py

```diff
@@ -2992,12 +2992,15 @@ def misdirect(player, alive_players=None):
     return random.choice([alive_players[len(alive_players)-1] if alive_players.index(player) == 0 else alive_players[alive_players.index(player)-1], alive_players[0] if alive_players.index(player) == len(alive_players)-1 else alive_players[alive_players.index(player)+1]])
 
 def has_privileges(level, message):
+    command = message.content.strip()[len(BOT_PREFIX):].lower().split(' ')[0]
```

Comment:
Hmm this is really janky, I think a better long term solution is adding an optional parameter `spec=False` to the @cmd decorator and doing the check in `on_message` instead.

Reply:
Good idea, I will get on it.